src/main/java/org/folio/rest/impl/TenantReferenceAPI.java

```diff
@@ -64,7 +67,8 @@ private boolean buildDataLoadingParameters(TenantAttributes tenantAttributes, Te
         .add("invoices", "invoice-storage/invoices")
         .add("invoice-lines", "invoice-storage/invoice-lines")
         .add("vouchers", "voucher-storage/vouchers")
-        .add("voucher-lines", "voucher-storage/voucher-lines");
+        .add("voucher-lines", "voucher-storage/voucher-lines")
+        .add("acquisitions-unit-assignments", "orders-storage/acquisitions-unit-assignments");
```

Comment:
I believe `orders-storage` should be renamed to `invoice-storage`

Reply:
fixed